src/operator-manual/metrics/01-quickstart.md

```diff
@@ -0,0 +1,235 @@
+Prometheus community helm chart.
+
+Let's add the helm repository from the Prometheus Community:
+
+```console
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+```
+
+Now, let's install the
+[`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+chart. This chart contains a collection of Kubernetes manifests, Grafana dashboards, and Prometheus
+rules.
+
+Let's create a `kube-prometheus-stack-values.yaml` file to configure the `kube-prometheus-stack`
+Helm chart values with the following contents:
+
+```yaml
+prometheus:
+  additionalServiceMonitors:
+    - name: kubewarden
+      selector:
+        matchLabels:
+          app: kubewarden-policy-server-default
+      namespaceSelector:
+        matchNames:
+          - kubewarden
+      endpoints:
+        - port: metrics
+          interval: 10s
+```
+
+The `prometheus-operator` deployed as part of this Helm chart defines the concept of [Service
+Monitors](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/design.md#servicemonitor),
+used to declaratively define what services should be monitored by Prometheus.
+
+In our case, we are adding a Service monitor targeting the `kubewarden` namespace, for services that
+match labels `app=kubewarden-policy-server-ha-policy-server`. This way, the Prometheus Operator is
+able to inspect what `endpoints` are tied to services matching this conditions. The operator will
+then take care of generating a valid configuration file for Prometheus, and reload Prometheus
+automatically after updating its configuration file.
+
+```console
+helm install --wait --create-namespace --namespace prometheus --values kube-prometheus-stack-values.yaml prometheus prometheus-community/kube-prometheus-stack
+```
+
+## Install Kubewarden
+
+We can now install Kubewarden in the recommended way with the Helm chart.
+
+> **Note well:** cert-manager and OpenTelemetry are a requirement of Kubewarden, but we've already
+> installed them in a previous section of this book.
+
+As a first step, we have to add the Helm repository that contains Kubewarden:
+
+```console
+helm repo add kubewarden https://charts.kubewarden.io
+```
+
+Then we have to install the Custom Resource Definitions (CRDs) defined by
+Kubewarden:
+
+```console
+helm install kubewarden-crds kubewarden/kubewarden-crds
+```
+
+Now we can deploy the rest of the Kubewarden stack. The official helm chart will create a
+PolicyServer named `default`. The default Helm chart values are already set up in a way that metrics
+are exposed for this default Policy Server. The relevant Helm Chart values are:
+
+```yaml
+policyServer
+  telemetry:
+    metrics:
+      port: 8080
+```
+
+Note that the default `values.yaml` for this Helm Chart are already set to this values. There is no
+need set this values if you don't want to change anything else.
```

Comment:
Just out of curiosity, is everything fine if the user doesn't have OpenTelemetry installed?

Reply:
Just tried out of curiosity.

If OpenTelemetry is not installed and on the `values.yaml` they provide:

```yaml
policyServer:
  telemetry:
    enabled: False
```

Everything is as expected. No unwanted behaviors anywhere I could see.

If OpenTelemetry is left enabled (default `values.yaml`), but OpenTelemetry is not installed, we get an installation error with Helm because the `OpenTelemetryCollector` resource kind is not registered in the API server. I think this is fine and desirable.

Reply:
should we default to OpenTelemetry disabled on the chart, for an easier first-use? (if not, we would need to change docs of charts, plus the docs page, for all deployments of kubewarden).

Reply:
I think we should default to telemetry enabled by default with Helm. Any kind of deployment on top of Kubernetes requires some visibility.

I am actually missing on this PR to update the quickstart where the user disables it. Adding that.

Reply:
Fair enough. Commented on the chart PR, that the chart upgrades would need a release notice.

Reply:
Updated the PR, thanks :)